README.md

```diff
@@ -0,0 +1,41 @@
+# GDX Analytics Looker Google MyBusiness Service BC
+
+GDX Analytics Looker project to integrate the Google MyBusiness API for Location Insights from Service BC locations
+
+## Features
```

Comment:
Can you rewrite this to describe this projects features?

Reply:
Done